lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -152,6 +152,23 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
       const estValuesEl = this.dom.createChildOf(metricAuditsEl, 'div', 'lh-metrics__disclaimer');
       const disclaimerEl = this.dom.convertMarkdownLinkSnippets(strings.varianceDisclaimer);
       estValuesEl.appendChild(disclaimerEl);
+
+      // Add link to score calculator
+      const calculatorLink = this.dom.createChildOf(estValuesEl, 'a');
+      calculatorLink.textContent = strings.calculatorLink;
+      const v5andv6metrics = /** @type {LH.ReportResult.AuditRef[]} */ ([
+        ...metricAudits,
+        category.auditRefs.find(m => m.id === 'first-cpu-idle'),
+        category.auditRefs.find(m => m.id === 'first-meaningful-paint'),
+      ]);
+      const metricPairs = v5andv6metrics.map(audit => {
+        const value = (audit.result.numericValue || 0).toString();
```

Comment:
wouldnt this make a super good score for errors?

what happens if we pass along a null instead?

Reply:
that
![image](https://user-images.githubusercontent.com/2301202/81699756-5e9ce080-942d-11ea-8aeb-a353b61601f8.png)


which actually isn't bad, something is obviously wrong and score goes to `0`, so I agree we pass `null` and update score calc to show a `?` or something when there's a NaN

Reply:
sg. null!